src/Tools/CompilerBenchmarks/run-perf.ps1

```diff
@@ -0,0 +1,36 @@
+﻿# Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+[CmdletBinding(PositionalBinding=$false)]
+param (
+    # By default, the Roslyn dir is expected to be next to this dir
+    [string]$roslynDir = "$PSScriptRoot/../../.."
+)
+
+Set-Variable -Name LastExitCode 0
+Set-StrictMode -Version 2.0
+$ErrorActionPreference = "Stop"
+
+try
+{
+    . "$roslynDir/build/scripts/build-utils.ps1"
+    $binariesDir = Resolve-Path "$roslynDir/Binaries"
```

Comment:
This is already defined in build-utils.p1s. No need to redefine it.

Reply:
Note: this probably doesn't do what you think it does. `Resolve-Path` does not return a `string`. 

---
In reply to: [210432872](https://github.com/dotnet/roslyn/pull/29323#discussion_r210432872) [](ancestors = 210432872)